dot/network/service.go

```diff
@@ -440,15 +440,23 @@ func (s *Service) handleConn(conn libp2pnetwork.Conn) {
 
 	hs, err := prtl.getHandshake()
 	if err != nil {
+		logger.Warnf("failed to get handshake for protocol %s: %s",
+			prtl.protocolID,
+			err,
+		)
 		return
 	}
 
-	stream, err := s.sendHandshake(conn.RemotePeer(), hs, prtl)
+	_, err = s.sendHandshake(conn.RemotePeer(), hs, prtl)
 	if err != nil {
+		logger.Debugf("failed to send handshake to peer on connection, peer=%s: %s",
```

Comment:
supernit making it more sentence oriented

```suggestion
		logger.Debugf("failed to send handshake to peer %s on connection: %s",
```

Reply:
done